index.html

```diff
@@ -692,6 +746,22 @@ <h2>Path&nbsp; and value</h2>
           specifications</li>
         <li>XML default namespace is assumed to be <code>http://www.w3.org/ns/ttml</code></li>
       </ul>
+      <p>The <code>ebuttm:</code> prefix is used for the URN <code>urn:ebu:tt:metadata</code>;
+        that namespace, and the <code>ebuttm:conformsToStandard</code> and
+        <code>ebuttm:documentMetadata</code>
+        elements are defined by [[!EBU-TT-M]].
+      </p>
```

Comment:
I would split the note into two notes since there are different concepts:

_In the definition above, the `ebuttm` prefix corresponds to the XML namespace `urn:ebu:tt:metadata`_

_the `ebuttm:conformsToStandard` and `ebuttm:documentMetadata` elements are defined in `[[!EBU-TT-M]]`_

Reply:
Thanks I'll adjust the wording. The text already says the namespace is defined in `[[!EBU-TT-M]]`.